packages/editor-ui/src/constants.ts

```diff
@@ -551,3 +551,11 @@ export const ALLOWED_HTML_TAGS = [
 ];
 
 export const GITHUB_STARS_BANNER_SHOW_UNTIL_DATE = new Date('2023-06-01');
+
+export const CHANGE_PLAN_PAGE = 'https://app.n8n.cloud/account/change-plan';
+
+export const CLOUD_BASE_URL_STAGING = 'https://stage-app.n8n.cloud';
+
+export const CLOUD_BASE_URL_PRODUCTION = 'https://app.n8n.cloud';
```

Comment:
these are wrong.. should be `api-`. I wonder whether we should add these to metadata.. in case we ever want to change the main host..